src/server/trpc/router/event/deleteEvent.ts

```diff
@@ -0,0 +1,25 @@
+import { protectedProcedure } from '~/server/trpc/trpc'
+import { z } from 'zod'
+import { toDataURL } from 'qrcode'
+import { randomUUID } from 'crypto'
+import { env } from '~/env/server.mjs'
+import eventCollection from '~/server/db/collections/EventCollection'
+import { Timestamp } from 'firebase/firestore'
+import userCollection from '~/server/db/collections/UserCollection'
+import logCollection from '~/server/db/collections/LogCollection'
+
+export const deleteEvent = protectedProcedure
+    .input(z.string())
+    .query(async ({ input }) => {
```

Comment:
Likewise, deleting an event is a `mutation` so you should be using `mutation` here instead of a `query`